.generator/src/generator/templates/configuration.j2

```diff
@@ -106,6 +107,10 @@ export interface ConfigurationParameters {
    * Default index of a server to use for an operation from the predefined server operation map
    */
   operationServerIndices?: { [ name: string ]: number };
+  /**
+   * Custom `fetch` function
+   */
+  fetch?: typeof crossFetch;
```

Comment:
I'm not sure whether this is the right type or I should just use `any` (for example https://github.com/octokit/request.js allows you to pass a custom `fetch` function but types it as `any`:  https://github.com/octokit/types.ts/blob/b21cc042affd15918abdc0636cb017e3751bbadc/src/Fetch.ts) because the Node fetch and browser fetch types aren't fully compatible: https://github.com/nodejs/undici/issues/1943

Reply:
In my opinion, `any` sounds a little safer to me.

Reply:
Agreed! Feel free to push edits to this PR directly if you have edit rights